src/renderer/stores/notebooks.js

```diff
@@ -57,6 +73,20 @@ var Notebooks = Backbone.Collection.extend({
     this.set(notebooks);
   },
 
+  handleCheckDeleteNotesReply: function(deleteStatus, id) {
+    this.destroyNotebook(id);
+    if (deleteStatus) {
+      console.log('deleting notes');
+      actionCreator.destroyNotesById(id);
```

Comment:
can you dispatch these actions from the action creator and then catch/handle them in here, as per the current setup?